protractor-typescript-cucumber/tmp/Config/config.js

```diff
@@ -0,0 +1,35 @@
+"use strict";
+/*
+The config folder includes all the configuration files
+This example config file displays the basic protractor-cucumber framework configuration
+ts-node compiler is needed for cucumberjs
+tags option for specific scenarios added
+**/
+exports.config = {
+    directConnect: true,
+    baseUrl: 'http://juliemr.github.io/protractor-demo/',
```

Comment:
it is the same change as above, typescript files are trans-compiled to javascript files!